src/components/MostRecentScoreboard.tsx

```diff
@@ -28,20 +52,45 @@ function MostRecentScoreboard(props: MostRecentScoresProps) {
   };
 
   return (
-    <div className={styles.grid}>
```

Comment:
@garrett-thompson 

I couldn't figure out how to make the buttons stay aligned and flush with the cards by tweaking `styles.grid`, so I gave up for now. Would love help!